tests/e2e/e2e.go

```diff
@@ -172,7 +172,7 @@ func setupSuite(c kubernetes.Interface, extClient versioned.Interface, apiExtCli
 		if localStorageSC == nil {
 			log.Fail("local-storage storage class not found")
 		}
-		if localStorageSC.Annotations == nil {
+		if localStorageSC != nil && localStorageSC.Annotations == nil {
```

Comment:
we already have `if localStorageSC == nil {` in L172?

Reply:
yes, but `gosec` still report this issue. 😂

Reply:
nolint?

Reply:
I think it's because log.Fail is not guaranteed to quit statically

Reply:
we can just return after L173

Reply:
got it